specification/contosowidgetmanager/Contoso.Management/tspconfig.yaml

```diff
@@ -21,6 +21,8 @@ options:
     package-dir: "azure-mgmt-contoso"
     package-name: "{package-dir}"
     generate-test: true
+    generate-sample: true
```

Comment:
How about the sample project for DPG? Should that be updated either?

Reply:
make sense